Library/Formula/smartmontools.rb

```diff
@@ -3,8 +3,8 @@
 class Smartmontools < Formula
   desc "SMART hard drive monitoring"
   homepage 'http://sourceforge.net/apps/trac/smartmontools/'
```

Comment:
This URL is no longer valid; would you update it to its permanent redirect (+https) ?:
`homepage "https://sourceforge.net/projects/smartmontools/"`